test/libsolidity/syntaxTests/license/license_cr_endings.sol

```diff
@@ -1 +1,2 @@
-// This test is actually useless, as the test suite adds the automatic preamble.
// SPDX-License-Identifier: GPL-3.0
contract C {}
\ No newline at end of file
+// This test is actually useless, as the test suite adds the automatic preamble.
// SPDX-License-Identifier: GPL-3.0
contract C {}
+// ----
```

Comment:
We do not want this one or the one for CR LF below. The file was specifically made to have only CR line endings (including the last one).